querybook/server/lib/metastore/loaders/glue_data_catalog_loader.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+from typing import Dict, List, Tuple
+
+from clients.glue_client import GlueDataCatalogClient
+from lib.form import StructFormField, FormField, FormFieldType
+from lib.metastore.base_metastore_loader import (
+    BaseMetastoreLoader,
+    DataTable,
+    DataColumn,
+)
+
+
+class GlueDataCatalogLoader(BaseMetastoreLoader):
+    def __init__(self, metastore_dict: Dict):
+        self.catalog_id = metastore_dict.get("metastore_params").get("catalog_id")
+        self.region = metastore_dict.get("metastore_params").get("region")
+        self.load_partitions = metastore_dict.get("metastore_params").get(
+            "load_partitions"
+        )
+        self.glue_client = self._get_glue_data_catalog_client(
+            self.catalog_id, self.region
+        )
+        super(GlueDataCatalogLoader, self).__init__(metastore_dict)
+
+    @classmethod
+    def get_metastore_params_template(cls):
+        return StructFormField(
+            catalog_id=FormField(
+                required=True,
+                description="Enter the Glue Data Catalog ID",
+                regex=r"^\d{12}$",
+            ),
+            region=FormField(required=True, description="Enter the AWS Region"),
+            load_partitions=FormField(
+                required=False,
+                description="Enter True or False",
```

Comment:
The description can be changed since you can only enter true or false

Reply:
Thanks for the hint.

I removed the description entirely, as it does not show up in the UI anyways.

![image](https://user-images.githubusercontent.com/9784890/119056429-f9c67f80-b9ca-11eb-9250-9a1fb9afa158.png)